chapters/annotations.tex

```diff
@@ -256,7 +256,8 @@ \subsubsection{Text Markup in Captions}\doublelabel{text-markup-in-captions}
 
 A sequence of one or more newlines (encoded either literally or using the \lstinline!\n!
 escape sequence) means a paragraph break.  (A line break within a paragraph is
-not supported.)
+not supported.)  A sequence of newlines before the first paragraph or after the last
```

Comment:
If all newlines are ignored, maybe you could say "Any newlines before the..."? This could be interpreted in a way where you need multiple newlines to ignore them.

Reply:
That's indeed better, I have updated the PR.